card/mod/standard/set/self/activity_toolbar_button.rb

```diff
@@ -2,13 +2,14 @@
 
 format :html do
```

Comment:
delete?

Reply:
later. There is already a PT reminder for this: https://www.pivotaltracker.com/story/show/163321969